README.md

```diff
@@ -387,7 +387,7 @@ const meta = extractMetadata(props);
 
 ## Developing
 
-Note that this does _NOT_ use jfrog, so if you were using that you'll need to `npm config set registry https://registry.npmjs.org`
+Note that this does _NOT_ use a non-npmjs registry, so if you were using that you'll need to `npm config set registry https://registry.npmjs.org`
```

Comment:
I would suggest "If you are using a non-npmjs registry, then you will need to run `npm ...`